### ISSUE TITLE
triggers inputs cancelling each other out in desktop applications with libgdx 1.9.3/1.9.4

### ISSUE THREAD

**Issue by Mechrophile** [libgdx/libgdx#4273]
#### Issue details

similar to #4067 .  issue present in ljgwapplications on desktop in windows 10

Both Left trigger and Right trigger are mapped to the 4th axis of the controller. LT is 4th axis positive, RT is 4th axis negative. 
- If you have both triggers held down, you have a net 4th axis value of 0, not the desired result of having both triggers at max value
- when you look at the value of the axis, e.g. -0.25, you have no way of knowing what combination of values the triggers have. for instance, to produce -0.25, LT could be 0.5 and RT could be -0.75, or LT could be 0.3 and RT could be -0.05. its impossible to know which.
#### Reproduction steps/code

Pressing only the LEFT trigger, getAxis(4) returns [0,1].
Pressing only the RIGHT trigger, getAxis(4) returns [0,-1].
Pressing BOTH triggers, getAxis(4) returns 0.
#### Version of LibGDX and/or relevant dependencies

LibGDX 1.9.4 or 1.9.3, Windows 10, Xbox 360 Wired Controller.
#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [x] Windows
- [ ] Linux
- [ ] MacOS

**Comment by MobiDevelop** [comment]
Are you using the lwjgl3 backend?

**Comment by Mechrophile** [comment]
Not when I posted the issue, it was a lwjgl application.  I have since switched to Lwjgl3 for precisely this reason.  The fix definitely made it into lwjgl3.  I have heard its a limitation of the API provided by windows for the 360 controller.  Somehow that limitation was worked around in lwjgl3, so I figured I would draw attention to the need to get that fix into lwjgl -- if its impossible, thats no problem, bug can be closed.  Like I said, I'm no longer experiencing the problem since I switched to 3
